### ISSUE TITLE
Valid match, but no snatch

### ISSUE THREAD

**Issue by piejanssens** [SiCKRAGETV/sickrage-issues#2636]
See http://pastebin.com/EXWVT8DX

Set at initial SDTV and replace with 720p HDTV.

PS. I'm missing the part in the debug log where it's logged why it's not a valid episode.

"The result Fifth.Gear.S25E06.PDTV.x264-C4TV doesn't seem to be a valid episode that we are trying to snatch, ignoring"

**Comment by miigotu** [comment]
@piejanssens a partial log doesn't help. Particularly that I cant see what your any,best line is, and what the existing episode status is.

`Fifth.Gear.S25E06.PDTV.x264-C4TV` is SDTV, and if you already have an SDTV episode then obviously another SDTV episode is not one that you are trying to snatch.

**Comment by piejanssens** [comment]
Cannot provide the full log at this time but the status is 'wanted' and there is no existing file.

**Comment by miigotu** [comment]
Then maybe the show didn't match one in your list exactly. I can't help without the full debug log.

**Comment by piejanssens** [comment]
here you go - http://pastebin.com/XUjPzeEn

**Comment by miigotu** [comment]
@piejanssens did you do a full update on the show after enabling xem? 

`2015-09-02 12:38:51 DEBUG    Thread-53 :: Nothing needs to be downloaded for Fifth Gear, skipping`

**Comment by piejanssens** [comment]
AFAIK I initially added the show with XEM enabled.

**Comment by piejanssens** [comment]
I did a full forced update anyway now.

I also noticed this:
```
2015-09-02 13:15:10 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: The Data returned from TorrentLeech do not contains any torrent
2015-09-02 13:15:07 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Search string: https://torrentleech.org/torrents/browse/index/query/Fifth.Gear%20S26E06%
```
If I paste that URL in my browser I get 400 - Bad Request.

**Comment by piejanssens** [comment]
SR is not taking the XEM into account anymore. It's now searching for S26E06 instead of S25E06.

```
2015-09-02 13:15:10 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: The Data returned from TorrentLeech do not contains any torrent
2015-09-02 13:15:07 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Search string: https://torrentleech.org/torrents/browse/index/query/Fifth.Gear%20S26E06%20/categories/2,26,27,32
2015-09-02 13:15:06 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: search_strings = [{'Episode': [u'Fifth.Gear S26E06 ', u'5th.Gear S26E06 ']}]
2015-09-02 13:15:06 INFO     SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Performing episode search for Fifth Gear
2015-09-02 13:15:06 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Transaction with 2 queries executed
```

**Comment by miigotu** [comment]
because that is correct.  toggle the xem the other way and force a full update and it should work.

**Comment by miigotu** [comment]
simple solution is to remove the show from SR, remove the tvshow.nfo|series.xml if you have it in the dir, and add it from existing but pick the tvrage result.

**Comment by piejanssens** [comment]
http://thexem.de/xem/show/2518

S25E06 is what SR should be searching for to get the latest episode.
XEM db is up to date and correct.

**Comment by miigotu** [comment]
I made no claim that XEM was wrong. I gave you a solution.

**Comment by piejanssens** [comment]
If XEM is correct and scene numbering is enabled then I can expect SR to search for S25E06 and not the tvdb numbering S26E06. This looks like a bug to me. 

I removed the show and readded it with tvdb as source + scene numbering enabled.
Now it is searching correcly again and we are back at the initial issue where SR is not picking up the hit although it should.

```
2015-09-02 15:14:02 INFO     SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Unable to find a download for: [Fifth Gear - 26x06 - Episode 6]
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: formatting pattern: %SN - %Sx%0E - %EN -> Fifth Gear - 26x06 - Episode 6
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Episode has no release name, replacing it with a generic one: %SN - %Sx%0E - %EN
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: formatting pattern: %SN - %Sx%0E - %EN -> Fifth Gear - 26x06 - Episode 6
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Episode has no release name, replacing it with a generic one: %SN - %Sx%0E - %EN
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Transaction with 1 queries executed
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Added RSS item: [Fifth.Gear.S25E06.PDTV.x264-C4TV] to cache: [torrentleech]
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Adding item from search to cache: Fifth.Gear.S25E06.PDTV.x264-C4TV
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: The result Fifth.Gear.S25E06.PDTV.x264-C4TV doesn't seem to be a valid episode that we are trying to snatch, ignoring
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Using cached parse result for: Fifth.Gear.S25E06.PDTV.x264-C4TV
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: The Data returned from TorrentLeech do not contains any torrent
2015-09-02 15:14:00 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Search string: https://torrentleech.org/torrents/browse/index/query/5th.Gear%20S25E06%20/categories/2,26,27,32
2015-09-02 15:14:00 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Found result: Fifth.Gear.S25E06.PDTV.x264-C4TV (https://torrentleech.org/download/637285/Fifth.Gear.S25E06.PDTV.x264-C4TV.torrent)
2015-09-02 15:13:58 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Search string: https://torrentleech.org/torrents/browse/index/query/Fifth.Gear%20S25E06%20/categories/2,26,27,32
2015-09-02 15:13:58 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: search_strings = [{'Episode': [u'Fifth.Gear S25E06 ', u'5th.Gear S25E06 ']}]
2015-09-02 15:13:57 INFO     SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Performing episode search for Fifth Gear
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: Last update was too soon, using old cache: 2015-09-02 15:05:25. Updated less then 20 minutes ago
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: Internal name cache for Fifth Gear set to: [ fifth gear, 5th gear ]
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: Building internal name cache for Fifth Gear
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: No scene exceptions update needed
2015-09-02 15:13:57 INFO     SEARCHQUEUE-MANUAL-78998 :: Beginning manual search for: [Fifth Gear - 26x06 - Episode 6]
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: formatting pattern: %SN - %Sx%0E - %EN -> Fifth Gear - 26x06 - Episode 6
2015-09-02 15:13:57 DEBUG    SEARCHQUEUE-MANUAL-78998 :: Episode has no release name, replacing it with a generic one: %SN - %Sx%0E - %EN
```

**Comment by miigotu** [comment]
So still not following what I asked you to do? ok, done helping.

**Comment by piejanssens** [comment]
You asked me to avoid the problem by using tvrage. 
I'd rather have this problem fixed.

**Comment by miigotu** [comment]
"toggle the xem the other way and force a full update and it should work."

**Comment by miigotu** [comment]
It is working correct, your understanding is not.

**Comment by piejanssens** [comment]
For the record I tried "toggle the xem the other way and force a full update and it should work." and it didn't work. I also removed it completely and added it as tvrage as you suggested, same problem.

```
2015-09-02 15:14:02 DEBUG    SEARCHQUEUE-MANUAL-78998 :: [TorrentLeech] :: The result Fifth.Gear.S25E06.PDTV.x264-C4TV doesn't seem to be a valid episode that we are trying to snatch, ignoring
```

There is something wrong with that part. It's searching for the correct episode if scene numbering is enabled, but then somehow it's not liking the search result that to me looks like a perfect match.
